spec/helpers/ems_cloud_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,46 @@
+require "spec_helper"
+
+describe EmsCloudHelper do
+  before do
+    controller.send(:extend, EmsCloudHelper)
+    self.class.send(:include, EmsCloudHelper)
```

Comment:
I more mean doing `self.class` in the before is including it into the example.  If controller is defined correctly, it should *already* have the EmsCloudHelper included.